.github/workflows/build.yml

```diff
@@ -268,3 +268,142 @@ jobs:
+          -DENABLE_GR_BLOCKS=ON \
+          -DENABLE_GR_DIGITAL=ON \
+          -DENABLE_GR_FFT=ON \
+          -DENABLE_GR_FILTER=ON \
+          -DENABLE_GR_NETWORK=ON
+        cmake --build build
+        cmake --install build
+    - name: Clone and build rtl-sdr
+      working-directory: ${{ runner.temp }}
+      run: |
+        git clone https://gitea.osmocom.org/sdr/rtl-sdr.git
+        cd rtl-sdr
+        cmake -S . -B build \
+          -DCMAKE_INSTALL_PREFIX=${RUNNER_TEMP}/msys64/mingw64 \
+          -DCMAKE_BUILD_TYPE=Release \
+          -DDETACH_KERNEL_DRIVER=OFF \
+          -DENABLE_ZEROCOPY=OFF \
+          -DINSTALL_UDEV_RULES=OFF
+        cmake --build build
+        cmake --install build
+    - name: Clone and build gr-osmosdr
+      working-directory: ${{ runner.temp }}
+      run: |
+        git clone https://gitea.osmocom.org/sdr/gr-osmosdr.git
+        cd gr-osmosdr
+        cmake -S . -B build \
+          -DCMAKE_INSTALL_PREFIX=${RUNNER_TEMP}/msys64/mingw64 \
+          -DCMAKE_BUILD_TYPE=Release \
+          -DENABLE_DEFAULT=OFF \
+          -DENABLE_FILE=ON \
+          -DENABLE_RTL=ON \
+          -DENABLE_SOAPY=ON
+        cmake --build build
+        cmake --install build
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Configure, compile & install
+      run: |
+        cmake -S . -B build \
+          -DCMAKE_INSTALL_PREFIX=C:/gqrx \
+          -DCMAKE_BUILD_TYPE=Release
+        cmake --build build
+        cmake --install build
+    - name: Deploy Qt application
+      working-directory: ${{ runner.temp }}/msys64/mingw64/bin
+      run: |
+        windeployqt6 C:/gqrx/bin/gqrx.exe
+        cp \
+          libb2-1.dll \
+          libboost_program_options-mt.dll \
+          libboost_thread-mt.dll \
+          libbrotlicommon.dll \
+          libbrotlidec.dll \
+          libbz2-1.dll \
+          libdouble-conversion.dll \
+          libfftw3f-3.dll \
+          libfftw3f_threads-3.dll \
+          libFLAC.dll \
+          libfmt.dll \
+          libfreetype-6.dll \
+          libgcc_s_seh-1.dll \
+          libglib-2.0-0.dll \
+          libgmp-10.dll \
+          libgnuradio-*.dll \
+          libgraphite2.dll \
+          libharfbuzz-0.dll \
+          libiconv-2.dll \
+          libicudt74.dll \
+          libicuin74.dll \
+          libicuuc74.dll \
+          libintl-8.dll \
+          libmd4c.dll \
+          libmp3lame-0.dll \
+          libmpg123-0.dll \
+          libogg-0.dll \
+          libopus-0.dll \
+          liborc-0.4-0.dll \
+          libpcre2-16-0.dll \
+          libpcre2-8-0.dll \
+          libpng16-16.dll \
+          librtlsdr.dll \
+          libsndfile-1.dll \
+          libSoapySDR.dll \
+          libspdlog.dll \
+          libstdc++-6.dll \
+          libusb-1.0.dll \
+          libvolk.dll \
+          libvorbis-0.dll \
+          libvorbisenc-2.dll \
+          libwinpthread-1.dll \
+          libzstd.dll \
+          zlib1.dll \
```

Comment:
I had to put together this list of library dependencies manually. I wonder if there's a tool that could do this automatically (like dylibbundler on macOS).

Reply:
can it be helpful? https://github.com/eyllanesc/69108420/blob/main/.github/workflows/test.yml

Reply:
> can it be helpful?

What specifically were you looking at? It doesn't look like that project has any shared library dependencies like Gqrx does.

Reply:
Maybe this? https://github.com/mpreisler/mingw-bundledlls